subsys/net/lib/rest_client/src/rest_client.c

```diff
@@ -164,6 +165,11 @@ static int rest_client_sckt_connect(int *const fd,
 	};
 	struct sockaddr *sa;
 	int proto = 0;
+	int64_t sckt_connect_start_time;
+	int64_t sckt_connect_time = 0;
+	int64_t dns_time = 0;
```

Comment:
Funny, I intentionally added 2nd parameter to avoid complaints about using the same one in two places for different timings :)
I'll change to one value.